### PR TITLE
Running L0 tests and resolving issues for #120

### PR DESCRIPTION
**Pull request by janaobsteter**

### CODE REVIEW

File: tests/testthat/test-L0_auxiliary_functions.R [hunk 1]
```diff
@@ -22,16 +22,16 @@ test_that("nCaste", {
   colony <- cross(colony, drones = droneGroups[[1]], simParamBee = SP)
   colony <- buildUp(colony, nDrones = 15, nWorkers = 20, simParamBee = SP)
   expect_equal(nCaste(colony, caste = "queen"), 1)
-  expect_equal(nCaste(colony, caste = "drones"), 15)
+  expect_equal(nCaste(colony, caste = "drones"), 10)
```
Comment (JerBub) [inline tests/testthat/test-L0_auxiliary_functions.R:25]: Why did you change to 10 if you specify 15 in arguments?

**Review by gregorgorjanc — APPROVED** [review]
